src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs

```diff
@@ -417,10 +419,12 @@ static unsafe void CopyToBuffer(Span<byte> destinationBuffer, List<QuicBuffer> s
         internal override void AbortRead(long errorCode)
         {
             ThrowIfDisposed();
+            Debug.Assert(errorCode != 0);
```

Comment:
What's wrong with 0? How do we know the user app doesn't use 0 as something relevant?

Reply:
I was under impression from talking to @nibanks that 0 should be used only for graceful shutdown. This is not important for goal of this PR so I took it out.